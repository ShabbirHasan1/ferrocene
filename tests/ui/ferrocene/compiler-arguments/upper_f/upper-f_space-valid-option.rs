// When -F is followed by a valid option, the lint is denied
//
// check-fail
// compile-flags: -F missing_docs

fn main() {} //~ ERROR

// ferrocene-annotations: um_rustc_F
