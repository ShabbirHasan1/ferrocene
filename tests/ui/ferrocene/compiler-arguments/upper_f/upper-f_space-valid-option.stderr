error: missing documentation for the crate
  --> $DIR/upper-f_space-valid-option.rs:6:1
   |
LL | fn main() {}
   | ^^^^^^^^^^^^
   |
   = note: requested on the command line with `-F missing-docs`

error: aborting due to previous error

